packages/story-editor/src/components/header/buttons/buttonWithChecklistWarning.js

```diff
@@ -114,22 +122,33 @@ function ButtonWithChecklistWarning({
     toolTip = TOOLTIP_TEXT_REVIEW;
   }
 
+  const publishText = useMemo(
+    () =>
+      hasFutureDate && status !== 'private'
+        ? __('Schedule', 'web-stories')
+        : __('Publish', 'web-stories'),
+    [hasFutureDate, status]
+  );
+
+  const text = canPublish
+    ? publishText
+    : __('Submit for review', 'web-stories');
   return (
-    <Tooltip title={toolTip} hasTail>
+    <Tooltip title={toolTip} popupZIndexOverride={11} hasTail>
```

Comment:
This is just a thought, since we have to pass in `popupZindexOverride={11}` for a few things specifically for this publish modal, maybe we should make it a const / enum to ensure we are always using the same zindex. (or at least using the correct value for a particular use case.  Definitely not a blocker.